tests/settings.py

```diff
@@ -1,39 +1,29 @@
 # -*- coding: utf8 -*-
 
+import django
+
 DEBUG = False
-USE_SQLITE = False
-
-try:
-    from .local import USE_SQLITE
-except ImportError:
-    pass
-
-if USE_SQLITE:
-    DATABASES = {
-        'default': {
-            'ENGINE': 'django.db.backends.sqlite3',
-            'NAME': ':memory:',
-        }
-    }
 
-else:
-    DATABASES = {
-        'default': {
-            'ENGINE': 'django.db.backends.postgresql_psycopg2',
-            'NAME': 'django_nopassword',
-        }
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': ':memory:',
     }
-
+}
 
 AUTH_USER_MODEL = 'tests.CustomUser'
 
 NOPASSWORD_LOGIN_CODE_TIMEOUT = 900
 
 INSTALLED_APPS = [
+    'django.contrib.admin',
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
 
+    'rest_framework',
```

Comment:
We should probably have tests without rest framework to verify that it is not necessary.

Reply:
Do you think it is ok to have a test case where settings are overridden or does it need to be a separate test project?

Reply:
I am thinking that we want to change the test settings to only add rest_framework if it is possible to import it and then add a tox environment without it. What do you think?